main.go

```diff
@@ -43,6 +44,59 @@ type buildpackRecord struct {
 	LastUpdatedAt string
 }
 
+type buildpackReleaseInfo struct {
+	BuildpackName    string
+	BuildpackVersion string
+	BuildpackURL     string
+}
+
+func getBuildpackReleaseURL(buildpackName string) string {
+	// Returns the release notes page for a given buildpack; if the buildpack is
+	// not found, returns an empty string.
+
+	// Map of all supported system buildpack releases in Cloud Foundry.
+	buildpackReleaseURLs := map[string]string{
+		"staticfile_buildpack":  "https://github.com/cloudfoundry/staticfile-buildpack/releases",
+		"java_buildpack":        "https://github.com/cloudfoundry/java-buildpack/releases",
+		"ruby_buildpack":        "https://github.com/cloudfoundry/ruby-buildpack/releases",
+		"dotnet_core_buildpack": "https://github.com/cloudfoundry/dotnet-core-buildpack/releases",
+		"nodejs_buildpack":      "https://github.com/cloudfoundry/nodejs-buildpack/releases",
+		"go_buildpack":          "https://github.com/cloudfoundry/go-buildpack/releases",
+		"python_buildpack":      "https://github.com/cloudfoundry/python-buildpack/releases",
+		"php_buildpack":         "https://github.com/cloudfoundry/php-buildpack/releases",
+		"binary_buildpack":      "https://github.com/cloudfoundry/binary-buildpack/releases",
+		"nginx_buildpack":       "https://github.com/cloudfoundry/nginx-buildpack/releases",
+		"r_buildpack":           "https://github.com/cloudfoundry/r-buildpack/releases",
+	}
+
+	// Note that for a specific release, you'll need to append
+	// /tag/<version_number> at the end, e.g.,
+	// https://github.com/cloudfoundry/python-buildpack/releases/tag/v1.7.45
+	// for the Python buildpack.
+
+	if buildpackReleaseURL, ok := buildpackReleaseURLs[buildpackName]; ok {
+		return buildpackReleaseURL
+	}
+
+	return ""
+}
+
+func parseBuildpackVersion(buildpackFileName string) string {
+	// Takes a buildpack file name and parses out the version number from it.
+	// Buildpack filenames currently look like this: python_buildpack-cflinuxfs3-v1.7.43.zip
+	// "v1.7.43" is the version in this case.
+
+	fileNameParts := strings.Split(buildpackFileName, "-")
```

Comment:
What happens if the `buildpackVersion` can't be parsed from the file name? For example, if the file in the release is changed to `python_buildpack_cflinuxfs3_v1.7.43.zip` for some reason. Asking because when the buildpack URL is constructed, it appends the base `/releases` URL with `/tag/` in anticipation of the `buildpackVersion`. This would result in an invalid URL and a 404.

Reply:
Maybe there is a way to add a check for this, and if the `buildpackVersion` can't be parsed, don't append the `/tag/` to the URL. The base `/releases` URL should still work, and would be helpful for the user.

Reply:
I was about to say that to a certain degree we just have to rely on the URL structure from GitHub, but this is a good means of falling back.  I'll work this in!

Reply:
This is also why I split out the functions like I did, to make it easy to handle changes in the future if the filenames or the URL structures change.

Reply:
All fixed up, thanks @mheadd!